packages/server/test/unit/screenshots_spec.coffee

```diff
@@ -470,25 +470,37 @@ describe "lib/screenshots", ->
   context ".getPath", ->
     it "concats spec name, screenshotsFolder, and name", ->
       screenshots.getPath({
-        specName: "examples$/user/list.js"
+        specName: "examples/user/list.js"
         titles: ["bar", "baz"]
-        name: "quux/lorem*"
+        name: "quux/lorem"
       }, "png", "path/to/screenshots")
       .then (p) ->
         expect(p).to.eq(
-          "path/to/screenshots/examples$/user/list.js/quux/lorem.png"
+          "path/to/screenshots/examples/user/list.js/quux/lorem.png"
         )
 
     it "concats spec name, screenshotsFolder, and titles", ->
+      screenshots.getPath({
+        specName: "examples/user/list.js"
+        titles: ["bar", "baz"]
+        takenPaths: ["a"]
+        testFailure: true
+      }, "png", "path/to/screenshots")
+      .then (p) ->
+        expect(p).to.eq(
+          "path/to/screenshots/examples/user/list.js/bar -- baz (failed).png"
+        )
+
+    it "sanitizes file paths", ->
```

Comment:
Part of the purpose of this test is to prove that characters that aren't allowed get sanitized. Can we put something in that's not supposed to be in a filename and ensure it gets removed?

Reply:
Just to be clear, I'm talking about the `sanitizes file paths` spec. (GitHub highlighted the lines above the test for some reason)

Reply:
The `*` and `...` chars are being removed from the filename, are there certain characters that'd be better to test here?

Reply:
Sorry, my bad. I misread the diff 😓